bodhi/tests/client/test___init__.py

```diff
@@ -310,6 +310,43 @@ def test_url_flag(self, send_request):
                 'csrf_token': 'a_csrf_token'})
         self.assertEqual(bindings_client.base_url, 'http://localhost:6543/')
 
+    @mock.patch('bodhi.client.bindings.BodhiClient.csrf',
+                mock.MagicMock(return_value='a_csrf_token'))
+    @mock.patch('bodhi.client.bindings.BodhiClient.send_request', autospec=True)
+    def test_existing_override_error_message(self, send_request):
+        """
+        Assert that the error message is provided if we try to save an existing override
+        """
+        exception_message = "Buildroot override for js-tag-it-2.0-1.fc25 already exists"
+        send_request.side_effect = bindings.BodhiClientException(exception_message)
+        runner = testing.CliRunner()
+
+        result = runner.invoke(
+            client.save_buildroot_overrides,
+            ['--user', 'bowlofeggs', '--password', 's3kr3t', 'js-tag-it-2.0-1.fc25'])
```

Comment:
Just a recommendation (feel free to ignore): I suggest adding a space here. I like to group my tests into three groups visually, using spaces to separate them. The first group is code that sets up the test (like you have above this line). The middle group is usually a 1-liner that runs the thing being tested (i.e., this line). And the last group is the assertions that the right thing happened. It's just a personal preference, and just a suggestion ☺